soc/riscv/riscv-ite/it8xxx2/Kconfig.defconfig.it81302bx

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022 ITE Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_IT81302_BX
+
+config SOC
+	default "it81302bx"
+
+config SOC_IT8XXX2_GPIO_GROUP_K_L_DEFAULT_PULL_DOWN
```

Comment:
Is this config option already used somewhere?

Reply:
This config option is declared in it8xxx2/Kconfig.soc:
https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/riscv/riscv-ite/it8xxx2/Kconfig.soc#L32
And it is used in the CrOS EC repository:
https://chromium.googlesource.com/chromiumos/platform/ec/+/refs/heads/main/zephyr/boards/riscv/it8xxx2/it8xxx2_defconfig#25